includes/rest-api/Controllers/Version2/class-wc-rest-product-variations-v2-controller.php

```diff
@@ -981,7 +981,7 @@ public function get_item_schema() {
 								'context'     => array( 'view', 'edit' ),
 							),
 							'value' => array(
-								'description' => __( 'Meta value.', 'woocommerce' ),
+								'description' => __( 'Meta value.', 'woocommerce-rest-api' ),
```

Comment:
I think we should keep this set to `woocommerce` domain, no?

Reply:
good catch! fixed